packages/tsconfig.options.json

```diff
@@ -16,6 +16,7 @@
     "resolveJsonModule": true,
     "sourceMap": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
What does this do?

Reply:
Uh was just testing, I should remove it.